README.md

```diff
@@ -117,6 +117,14 @@ using map keys or 0-based array indexes separated by `/`.
     MaxMindDBEnv COUNTRY_CODE COUNTRY_DB/country/iso_code
     MaxMindDBEnv REGION_CODE  CITY_DB/subdivisions/0/iso_code
 
+### `MaxMindDBNetworkEnv` ###
```

Comment:
One more thought: it appears that you can only set `MaxMindDBNetworkEnv` once per database. This seems fine, but maybe we should document that as someone is going to try setting it multiple times.